core/includes/image.inc

```diff
@@ -97,6 +97,30 @@ function image_toolkit_invoke($method, stdClass $image, array $params = array())
   return FALSE;
 }
 
+/**
+ * Gets supported image extensions.
+ *
+ * Backdrop supports GIF, JPG, PNG, and WEBP file formats when supported by the
+ * GD toolkit, and may support others, depending on the currently used toolkit.
+ *
+ * @return array
```

Comment:
Looking at the code, that should be `array|bool` or `array|false`, as the function clearly can also return that.

Otherwise, maybe it should always return an array - empty if nothing's supported .